tests/test_program.py

```diff
@@ -828,3 +829,36 @@ def test_unsaved(self):
         with self.assertRaisesRegex(FaultError, "memory not saved in core dump") as cm:
             prog.read(0xFFFF0000, len(data) + 4)
         self.assertEqual(cm.exception.address, 0xFFFF000C)
+        self.assertEqual(prog.read(0xFFFF0000, len(data) + 4), data + bytes(4))
```

Comment:
FYI all of those test failures are coming from this line, which I think was a merge conflict resolution gone wrong (this line was removed by 02912ca7d07314597172970d0510d57d0d3574ba).

Reply:
That resolves one of the failures, the other one is:

```
test_dentry_path (tests.helpers.linux.test_fs.TestFs)
```

But I see that is failing on `main`, so I assume that has nothing to do with my changes.

Reply:
Ah yeah, that test fails when the current directory is under a bind mount, I need to fix that. It's not failing in the CI environment. Instead, it seems like Linux 4.19 is segfaulting.